frontend/src/components/buttons/ButtonDelete.vue

```diff
@@ -1,15 +1,18 @@
 <template>
   <v-btn
     :text="text"
-    class="px-3 px-sm-4"
+    class="ec-btn-delete px-3 px-sm-4"
     min-width="0"
     :color="color"
     :icon="iconOnly"
     v-bind="$attrs"
     v-on="$listeners"
   >
     <v-icon v-if="iconOnly">{{ icon }}</v-icon>
-    <v-icon v-else :left="$vuetify.breakpoint.smAndUp" size="150%">{{ icon }}</v-icon>
+    <v-icon v-else-if="!text" :left="$vuetify.breakpoint.smAndUp">{{ icon }}</v-icon>
```

Comment:
General question to all the changes to the Button components: As we use these components and quite many places, I'd assume you did some sanity check that layout elsewhere is not broken, correct?

Reply:
Yes, I could not spot a problem. Most of the UIs with buttons will be changed in my future camp admin PR anyway.